modules/economy-system/configs/strings.json

```diff
@@ -0,0 +1,88 @@
+{
+    "filename": "strings.json",
+    "content": [
+      {
+        "field_name": "notFound",
+        "default": "Cannot find the item",
+        "type": "string",
+        "description": "The message that is send if the item wasn't found",
+        "allowEmbed": false
```

Comment:
would be great if embeds were supported everywhere, as they are one of the most loved features (idk why either)